dependencies/src/test/java/com/google/cloud/tools/opensource/classpath/ClassFileTest.java

```diff
@@ -21,27 +21,28 @@
 import com.google.common.testing.EqualsTester;
 import com.google.common.testing.NullPointerTester;
 import com.google.common.testing.NullPointerTester.Visibility;
+import java.io.IOException;
 import java.nio.file.Paths;
 import org.junit.Test;
 
 public class ClassFileTest {
   @Test
-  public void testCreation() {
+  public void testCreation() throws IOException {
```

Comment:
It looks strange to throw IOException just to create instances.

Reply:
fixed